source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -416,7 +416,7 @@ Ubuntu as our working example.
 
       sudo clr-boot-manager set-timeout 25
 
-.. _download the live desktop image: https://clearlinux.org/downloads
+.. _download the live desktop image: https://clearlinux.github.io/downloads
 
-.. _Downloads: https://clearlinux.org/downloads
+.. _Downloads: https://clearlinux.github.io/downloads
```

Comment:
if it starts with download.clearlinux it doesn't change.  If it starts with clearlinux.org it changes.  That is the correct link but it's still having issues because of the new cname that got put in clearlinux.github.io.

Reply:
Gotcha, makes sense.